src/main/java/pl/fracz/mcr/comment/AbstractComment.java

```diff
@@ -0,0 +1,52 @@
+package pl.fracz.mcr.comment;
+
+import pl.fracz.mcr.preferences.ApplicationSettings;
+import pl.fracz.mcr.source.SourceFile;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class AbstractComment {
```

Comment:
nazwa AbstractComment sugeruje że to powinien być `abstract class`
jeżeli nie, lepiej moż DefaultComment